binary-array-ld-lib/src/main/kotlin/net/bald/model/ModelAliasDefinition.kt

```diff
@@ -35,10 +32,44 @@ class ModelAliasDefinition(
         }
     }
 
+    private fun identifyProperties(identifier: String): Sequence<Resource> {
+        return identifyResources(identifier).filter { resource ->
+            resource.hasProperty(RDF.type, RDF.Property) || resource.hasProperty(RDF.type, OWL.ObjectProperty)
+        }
+    }
+
     private fun identifyResources(identifier: String): Sequence<Resource> {
         return model.listResourcesWithProperty(DCTerms.identifier, identifier).asSequence()
     }
 
+    override fun isReferenceProperty(prop: Property): Boolean {
+        return prop.inModel(model).let { modelProp ->
+            modelProp.hasProperty(RDFS.range, BALD.Subject)
```

Comment:
I think `bald:Subject` is deprecated, and should not be used

Reply:
All of the uses of `bald:Subject` have been replaced.